src/snowflake/cli/api/project/schemas/streamlit/streamlit.py

```diff
@@ -28,15 +29,15 @@ class Streamlit(UpdatableModel, ObjectIdentifierModel(object_name="Streamlit")):
     query_warehouse: str = Field(
         title="Snowflake warehouse to host the app", default="streamlit"
     )
-    main_file: Optional[str] = Field(
+    main_file: Optional[Path] = Field(
```

Comment:
Why not SecurePath?

Reply:
Forgot we have it

Reply:
We use Path for models in many places. We can change it in future PRs.